types/index.d.ts

```
@@ -1046,163 +1046,231 @@ export declare function assert(received: any): Assertion;
+	 * @param {Error} error - The error to be thrown.
+	 * @returns {TrackFn} The updated tracked function.
+	 * @example trackFn.throw(new Error('Something went wrong'));
+	 */
+	throw(error: Error): TrackFn;
+	/**
+	 * Replaces the tracked function with a custom implementation.
+	 * @param {Function} fn - The custom function to use.
+	 * @returns {TrackFn} The updated tracked function.
+	 * @example trackFn.use((arg1, arg2) => arg1 + arg2);
+	 */
+	use(fn: Function): TrackFn;
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)